keyvault/2016-10-01/swagger/keyvault.json

```diff
@@ -2547,7 +2547,7 @@
           "type": "array",
           "items": {
             "type": "string",
-            "description": "JSON web key operations. For more information, see JsonWebKeyOperation.",
+            "description": "JSON web key operations. The default is all operations. For more information, see JsonWebKeyOperation.",
```

Comment:
@BrucePerlerMS yes default is all the values, which is "[ "encrypt", "decrypt", "sign", "verify", "wrapKey", "unwrapKey" ]".